### ISSUE TITLE
No demo page or screenshots

### ISSUE THREAD

**Issue by kmgilbert100** [fixate/angular-tourist#1]
Can you provide a demo page or screenshots so I can see whats going on before I take time to test it out?

**Comment by sdbondi** [comment]
We definitely need a demo - unfortunately we don't have time until maybe next year (PR's welcome). I'll give you the site we use it in for you to take a look. Once you start a tasting click "start tour" - https://goo.gl/XMFmGx

**Comment by kmgilbert100** [comment]
Wow, I definitely like the example site ill try it out let you know what I think... And as long as your active I am sure I will submit some PRs. - Is all the code committed right now the latest and greatest version like your using there?
- Best Regards

**Comment by sdbondi** [comment]
Yup :) Using v0.1.11 (latest) using the angular-tourist directives and configuration api to create that tour.
We are using qTip for the tooltips.

A simple example page that we can improve on if necessary would go far.
Let me know if you decide you want to create one and if you need any help. :+1:
